lib/internal/Magento/Framework/ObjectManager/Test/Unit/Config/_files/valid_config.xml

```diff
@@ -7,7 +7,7 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <preference for="Some_For_Name" type="Some_Type_Name" />
-    <virtualType name="" type="" shared="true">
```

Comment:
It is my understanding that these attributes can not be empty. I would like someone else to confirm or decline my understanding.